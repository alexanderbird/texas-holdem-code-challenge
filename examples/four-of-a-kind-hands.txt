6H 6C 6S 6D TC
Irvin TS TH
Klara JC 2C
Alyson 9C JD
Ulberto 9H QH
Cheater 7C 7S 7H 7D
