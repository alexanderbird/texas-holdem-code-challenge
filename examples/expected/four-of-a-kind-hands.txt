1 Cheater Four of a Kind 7
2 Ulberto Four of a Kind 6 (kicker Queen)
4 Klara Four of a Kind 6
4 Alyson Four of a Kind 6
5 Irvin Four of a Kind 6 (kicker 10)
